### DISCUSSION TITLE
Expose all parts of StorybookUI to allow re-arrangement

### DISCUSSION THREAD

**Discussion by pke** [storybookjs/react-native#388] [Ideas]
Would be nice if we had more control over how the UI is rendered.
I'd like to see the Addons below the preview and not beside it (which squeezes it too much).

**Comment by dannyhw** [comment]
This is I want to experiment with once v6 is stable, actually i've been thinking about making an option to have addons be a bottom sheet.

**Reply by pke** [reply]
I already coded it for 5.3 last night. Works pretty smooth. 
I will replace my current hand made "playground" with your storybook components and report back.
